db/migrate/20200204232207_add_shelters_to_reviews.rb

```diff
@@ -0,0 +1,5 @@
+class AddSheltersToReviews < ActiveRecord::Migration[5.1]
```

Comment:
generated in CL with `rails g migration AddSheltersToReviews shelter:references`